.github/workflows/test_iasworld_data.yaml

```diff
@@ -0,0 +1,141 @@
+        with:
+          role-to-assume: ${{ secrets.AWS_IAM_ROLE_TO_ASSUME_ARN }}
+
+      - name: Restore dbt state cache
+        id: cache
+        uses: ./.github/actions/restore_dbt_cache
+        with:
+          path: ${{ env.PROJECT_DIR }}/${{ env.STATE_DIR }}
+          key: ${{ env.CACHE_KEY }}
+
+      - name: Install Python dependencies
+        working-directory: ${{ env.PROJECT_DIR }}
+        shell: bash
+        run: uv pip install ".[dbt_tests]"
+
+      - name: Run tests
+        run: |
+          DEFER_OPTION=""
+          if [[ "$CACHE_HIT" == 'true' ]]; then
+            DEFER_OPTION="--defer --state $STATE_DIR"
+          fi
+          # shellcheck disable=SC2086
+          python scripts/run_iasworld_data_tests.py \
+            --target "$TARGET" \
+            --output-dir ./qc_test_results/ \
+            $SELECT_OPTION \
+            $SKIP_ARTIFACTS_OPTION \
+            $DEFER_OPTION
+        working-directory: ${{ env.PROJECT_DIR }}
+        shell: bash
+        env:
+          USER: ${{ github.triggering_actor }}
+          GIT_SHA: ${{ github.sha }}
+          GIT_REF: ${{ github.ref_name }}
+          GIT_AUTHOR: ${{ github.event.commits[0].author.name }}
+          CACHE_HIT: ${{ steps.cache.outputs.cache-hit }}
+          SELECT_OPTION: ${{ steps.parse-inputs.outputs.select-option }}
+          SKIP_ARTIFACTS_OPTION: ${{ inputs.upload_test_results && '--no-skip-artifacts' || '--skip-artifacts' }}
+
+      - name: Save test results to S3
+        if: inputs.upload_test_results
+        run: |
+          s3_prefix="s3://ccao-data-warehouse-us-east-1/qc"
+          local_prefix="qc_test_results/metadata"
+          for dir in "test_run" "test_run_result" "test_run_failing_row"; do
+            dirpath="${local_prefix}/${dir}"
+            if [ -e "$dirpath" ]; then
+              echo "Copying ${dirpath} metadata to S3"
+              aws s3 sync "$dirpath" "${s3_prefix}/${dir}"
+            fi
+          done
+
+          crawler_name="ccao-data-warehouse-qc-crawler"
+          aws glue start-crawler --name "$crawler_name"
+          echo "Triggered Glue crawler $crawler_name"
+        working-directory: ${{ env.PROJECT_DIR }}
+        shell: bash
+
+      - name: Get current time
+        if: failure()
+        run: echo "TIMESTAMP=$(date -u +"%Y-%m-%dT%H:%M:%S")" >> "$GITHUB_ENV"
+        shell: bash
+
+      # Only triggered when called by a bot. Otherwise, notifications
+      # go to whoever called the workflow
+      - name: Send failure notification
+        if: github.event_name == 'workflow_dispatch' && github.triggering_actor == 'sqoop-bot[bot]' && failure()
+        uses: ./.github/actions/publish_sns_topic
+        with:
+          sns_topic_arn: ${{ secrets.AWS_SNS_NOTIFICATION_TOPIC_ARN }}
+          subject: "iasWorld tests errored for workflow run: ${{ github.run_id }}"
+          body: |
+            iasWorld tests raised an error for workflow ${{ github.run_id }}, run on ${{ env.TIMESTAMP }} UTC
```

Comment:
This subject and body are now slightly different from the old implementation ("iasWorld tests" instead of "dbt tests") to make it clear that the message specifically relates to iasWorld tests.